### PR TITLE
Increase the AppArmor pod stop timeout to match the start timeout

### PR DESCRIPTION
**Pull request by timstclair**
Fixes https://github.com/kubernetes/kubernetes/issues/30750

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/31314)

### CODE REVIEW

**Comment by dchen1107** [comment]
LGTM

File: test/e2e_node/apparmor_test.go [hunk 1]
```diff
@@ -146,7 +146,9 @@ func loadTestProfiles() error {
 
 func runAppArmorTest(f *framework.Framework, profile string) api.PodStatus {
 	pod := createPodWithAppArmor(f, profile)
-	framework.ExpectNoError(f.WaitForPodNoLongerRunning(pod.Name))
+	// The pod needs to start before it stops, so wait for the longer start timeout.
```
Comment (yujuhong) [inline test/e2e_node/apparmor_test.go]: Just for my own education, all apparmor tests expect the pod to fail in certain way?

Comment (timstclair) [inline test/e2e_node/apparmor_test.go]: not necessarily fail. The pod container just runs touch and exits, and then the apparmor tests look at the container exit code to determine whether it succeeded (which should be conditional on whether the profile was applied correctly)

Comment (yujuhong) [inline test/e2e_node/apparmor_test.go]: I see. So the test expects the pod to terminate soon. Thanks for answering!

**Pull request merged**